030-Concepts/080-pricing.mdx

```diff
@@ -30,5 +30,47 @@ The data size on disk denotes the quantity of data held within the transactional
 This size limit is considered a soft-limit and is not strictly enforced in real-time. However, if you exceed this limit, it indicates that your data storage requirements may be approaching a critical level. In such cases, we recommend you explore potential solutions with [Xata Support](https://support.xata.io/hc/en-us) to optimize and scale your data storage.
 
 
+### FAQ
+
+#### What is a capacity unit?
+
+In most cases, you can think of a unit as 250,000 records. There are extra limits applied (rate limit, concurrency limit, data size limit) but with normal usage patterns you shouldn't hit them.
+
+#### Is there a maximum number of reads and writes per month?
+   
+No, you can serve as many reads and writes as you want within the rate limit, which is defined per second (resets every second).'
+  
+#### How is the data size measured?
+  
+We measure one copy of your data, as it is stored in the primary transactional store and file attachments. For data redundancy purposes, and because we use multiple types of stores, we store multiple copies of your data (typically 5 copies), but we only charge for one.'
```

Comment:
This is a bit different now. Storage is measured very differently than before. We now measure by database storage, search engine storage, file attachment storage, etc.